testlib/dbus.py

```diff
@@ -499,6 +499,28 @@ def pool_get_metadata(pool_path, *, current=True):
         )
         return iface.Metadata(current, timeout=StratisDbus._TIMEOUT)
 
+    @staticmethod
+    def fs_get_metadata(pool_path, *, fs_name=None, current=True):
+        """
+        Get filesystem-level metadata
+        :param str pool_path: The object path of the pool
+        :param str fs_name: The name of the filesystem
+        :param bool current: Current or most recently written metadata
+        :return: JSON-format pool-level metadata
+        :rtype: The D-Bus types s, q, and s
+        :raises dbus.exceptions.DBusException:
+        """
+        iface = dbus.Interface(
+            StratisDbus._BUS.get_object(StratisDbus._BUS_NAME, pool_path),
+            StratisDbus._POOL_IFACE,
+        )
+
+        fsname_spec = (False, "") if fs_name is None else (True, fs_name)
+
```

Comment:
No need to have a separate ```fs_name``` identifier, since you use the value only once.

Reply:
Resolved.

Reply:
You can condense it more to be consistent with other code, so just put the whole expression, ```(False,  ...) ...(True, fs_name)``` in the place of the ```fs_name``` argument in the FilesystemMetadata function call.